GameStore.Domain/Entities/Cart.cs

```diff
@@ -0,0 +1,13 @@
+﻿using GameStore.Domain.Common;
+
+namespace GameStore.Domain.Entities;
+
+public class Cart : BaseEntity
+{
+    public int UserId { get; set; }
+    public User User { get; set; }
+
+    public bool IsOrdered { get; set; }
+
+    public List<OrderItem> Items { get; set; } = new List<OrderItem>();
```

Comment:
Can we get rid of the separation of Order and Cart? And then we get rid of the extra table and service. The concept of Cart can be represented on the frontend side.

Reply:
Initially cart was stored only on frontend part, but then you requested for it to be stored in db so it can be shared beetween multiple sessions/devices.